apps/test/integration/levelSolutions/eval/display.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
   skinId: 'eval',
```

Comment:
This test was running into a locale-specific rendering difference in Chrome, where "1234" gets rendered as "1,234" in the code path under test, but not in the code path used to generate a block to pixel-compare to this value.

Fixed by using "123" instead of "1234" in these eval test cases to avoid the issue altogether. I don't feel too bad about changing our test to avoid this because we're not actively supporting `eval` anymore.